tools/PAMPA/pampa_macros.xml

```diff
@@ -67,6 +69,11 @@
     <xml name="pampa_output_GLM">
         <data name="output_recap" from_work_dir="GLMSummaryFull.txt" format="txt" label="Simple statistics on chosen variables on ${on_string}"/>
         <data name="output_rate" from_work_dir="RatingGLM.txt" format="txt" label="Your analysis rating file on ${on_string}"/>
+        <collection type="list" name="output_GLM">
```

Comment:
can we add a test for this outpu?

Reply:
It is an optional output hence I didn't used it for the test but I'll do it now ! Thank you !